homeassistant/components/system_log/__init__.py

```diff
@@ -165,7 +165,7 @@ def emit(self, record):
         """Emit a log record."""
         try:
             self.enqueue(record)
-        except asyncio.CancelledError:
+        except asyncio.CancelledError:  # pylint: disable=try-except-raise
```

Comment:
You can remove this cases completely. They are no longer caught by `Exception` since 3.8:

https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.CancelledError